packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -463,6 +480,8 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
                 containerWidth={containerWidth}
                 /* @conditional-compile-remove(PSTN-calls) */
                 onClickShowDialpad={alternateCallerId ? onClickShowDialpad : undefined}
+                /* @conditional-compile-remove(video-background-effects) */
+                onShowVideoEffectsOptions={setShowVideoEffectsPane}
```

Comment:
Can this just be?
```
onShowVideoEffectsOptions={() =>setShowVideoEffectsPane(true)}
```
given this modification https://github.com/Azure/communication-ui-library/pull/2825/files#r1144070999

https://github.com/Azure/communication-ui-library/pull/2825/files#r1144070999